src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResourceVersion2.java

```diff
@@ -287,6 +295,57 @@ public Response uploadIrbDocument(
     }
   }
 
+  @POST
+  @Consumes(MediaType.MULTIPART_FORM_DATA)
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("/progress_report/{parentReferenceId}")
+  @RolesAllowed({RESEARCHER})
+  public Response postProgressReport(
+      @Auth AuthUser authUser,
+      @PathParam("parentReferenceId") String parentReferenceId,
+      @FormDataParam("dar") String dar,
+      @FormDataParam("collaboratorRequiredFile") InputStream collabInputStream,
+      @FormDataParam("collaboratorRequiredFile") FormDataContentDisposition collabFileDetails,
+      @FormDataParam("ethicsApprovalRequiredFile") InputStream ethicsInputStream,
+      @FormDataParam("ethicsApprovalRequiredFile") FormDataContentDisposition ethicsFileDetails) {
+    User user = userService.findUserByEmail(authUser.getEmail());
+    DataAccessRequest parentDar = dataAccessRequestService.findByReferenceId(parentReferenceId);
+    checkAuthorizedUpdateUser(user, parentDar);
+    DataAccessRequest payload = populateDarFromJsonString(user, dar);
+    DataAccessRequest childDar = dataAccessRequestService.createDataAccessRequest(user, payload);
+
+    for (Integer datasetId : childDar.getDatasetIds()) {
+      Dataset dataset = datasetService.findDatasetById(datasetId);
+      DataUse dataUse = dataset.getDataUse();
```

Comment:
Connor and I found that `getDataUse()` can return null for about 25 or so datasets on dev

Reply:
What should the behavior be if `getDataUse` is null?

Reply:
I think we need to throw a server error so we can follow up. This isn't a user problem, it's an us problem in that we have a bad dataset that needs work.